### DISCUSSION TITLE
Issue with installing the janssen in the Ubuntu 20.04.x OS

### DISCUSSION THREAD

**Discussion by AmanNegi144** [JanssenProject/jans#5356] [Q&A] [answered]
Hi Everyone,
I am trying to install janssen 1.0.14 version in the Ubuntu 20.04.6 version. All the installation steps are getting completed but after the installation none of the jans services are running. Below is the screen shot of it.

![All the jans service status](https://github.com/JanssenProject/jans/assets/53539013/78c0c194-d19f-4539-94e2-80960efc48d4)

Along with it I am unable to start any of the services .
![when trying to start jans services](https://github.com/JanssenProject/jans/assets/53539013/381318c4-e3c6-4d8e-9c03-6cc934a5761d)

What could be the possible reason for it?

**Comment by mmrraju** [comment]
Hello Aman, What's about your DB? Please check your db status. Besides confirm that you have minimum [requirements](https://docs.jans.io/v1.0.13/admin/install/vm-install/vm-requirements/) for running all services.

**Reply by AmanNegi144** [reply]
As i have already mentioned we are using  20.04.6 version with 8 CPU, 16 GB of RAM and 125 GB of Disk space. You can check the details from below images 
![CPU](https://github.com/JanssenProject/jans/assets/53539013/58104048-acf5-4dd2-af15-1366d1487096)

![RAM](https://github.com/JanssenProject/jans/assets/53539013/8ae553c3-2e4a-44f7-9692-f818cd9d001f)

![Disk space](https://github.com/JanssenProject/jans/assets/53539013/a0d0a338-0439-480a-af18-7aac97824bbc)

And also I have gone with a local opendj installation option and here is the status of opendj service 
![Opendj](https://github.com/JanssenProject/jans/assets/53539013/7d9d6349-93a2-4540-a70e-fbf36d9ffa3c)

Please let me know if you require any more info.

**Reply by mmrraju** [reply]
So db status is ok. And you already try to restart all services but failed to load. 
Please let me some time to check from my end.

**Reply by mmrraju** [reply]
Hello Aman, I have just now tested from my end following [docs](https://docs.jans.io/v1.0.14/admin/install/vm-install/ubuntu/#ubuntu-2004). It's ok all service are running. Could you please try once more fresh installation?

**Reply by AmanNegi144** [reply]
Hi Mostafejur,
I just did the fresh install but still facing the same issue.

![Here is the image](https://github.com/JanssenProject/jans/assets/53539013/326766f2-7a28-41d1-8203-624005427620)

not sure how to proceed forward with this as I am not able to install the janssen in RHEL also.

**Reply by mmrraju** [reply]
Could you please follow some steps:
1.  check your ldap logs to find out what's wrong there? `/opt/opendj/logs`. If ldap messed up services can't read data.
2. Check persistence logs for all services to confirm that it's connected with ldap. i.e `/opt/jans/jetty/jans-auth/logs/jans-auth_persistence.log`

**Reply by AmanNegi144** [reply]
Hi Mostafejur,

1. The OpenDJ logs are clear can't find any error their.
![image](https://github.com/JanssenProject/jans/assets/53539013/88cc5334-8bff-4cfc-a9d8-8a41ace77b5c)


2. But the /opt/jans/jetty/jans-auth/logs/ folder is empty , i think because service never got started so no logs are generated here.

**Reply by AmanNegi144** [reply]
Hi @mmrraju ,

Is there any update on this?
Any solution through which we can use the janssen?

**Comment by ossdhaval** [comment, marked as answer]
Adding @devrimyatar for comments.

**Reply by devrimyatar** [reply]
What is your computer resources? If you are running on non-ssd and/or slow CPU, this is expected.

**Reply by ossdhaval** [reply]
@devrimyatar [Here](https://github.com/JanssenProject/jans/discussions/5356#discussioncomment-6318627) is the CPU config.
@AmanNegi144 are you on SSD or HDD?

**Reply by AmanNegi144** [reply]
Hi Dhaval,

We are using HDD currently but its a fresh Ubuntu server , so i don't think resources or slow PC is the issue.
below is the image of CPU and memory utilization in the server (as you can see its quite low)
![image](https://github.com/JanssenProject/jans/assets/53539013/acd7a446-4170-4bc0-9931-3e005771a5f5)

**Reply by AmanNegi144** [reply]
Hi @ossdhaval  ,

Is there any update on this?
Any solution through which we can use the janssen?

**Comment by mzico** [comment]
@AmanNegi144 : I took the liberty and tested as well. Couldn't reproduce the issue. 

Here are two recorded screen casts below. I took a fresh Ubuntu 20 VM with "Production" type resources. 

- https://youtu.be/XG9_malMwuU

- https://youtu.be/1mpTXNDCbGg

**Comment by ossdhaval** [comment]
@AmanNegi144 
We also fixed an issue that **_may_** change what you experienced: #5447. 
Since this fix is not yet released, you can use [dynamic install](https://docs.jans.io/head/admin/install/vm-install/dynamic-download/) to install the latest Janssen Server with this fix.
